### PR TITLE
fix comparing-and-sorting-tuples answer

### PR DESCRIPTION
**Pull request by borntofrappe**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

In the **Python for Everybody**, the [**Comparing and Sorting Tuples**]( https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples) challenge doesn't provide the right answer. The first one is the closest, but is lacking the `reverse=True` flag to match the input snippet.

```diff
-print( sorted( [ (v,k) for k,v in counts.items() ] ) )
+print( sorted( [ (v,k) for k,v in counts.items() ], reverse=True ) )
```

https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ question:
   answers:
     - |
         ```python
-        print( sorted( [ (v,k) for k,v in counts.items() ] ) )
+        print( sorted( [ (v,k) for k,v in counts.items(), reverse=True ] ) )
```
Comment (Sky020) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md:34]:
```suggestion
        print( sorted( [ (v,k) for k,v in counts.items() ], reverse=True ) )
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

**Comment by Sky020** [comment]
Thanks, for pointing this out, and getting a PR. Just a minor detail ⬆️

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md (hunk 1 above)
Comment (borntofrappe) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md:34]: Thanks for catching that. I feel a bit dumb...I created a small demo on repl.it to replicate the sorting, but messed up anyway.

**Review by ShaunSHamilton — DISMISSED** [review]
LGTM 👍

**Review by Manish-Giri — DISMISSED** [review]

**Comment by borntofrappe** [comment]
While working on a separate issue (#39096), I accidentally merged the commits in this pull request

https://github.com/freeCodeCamp/freeCodeCamp/issues/39096

First off, I'm just sorry for this mess 😢 
f you have any suggestion on how to fix this mess, I'll follow them to the letter. Sorry again. It seems I need to review how I work with git

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
@borntofrappe Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

Don't worry about the git mess, it's all good.  I would recommend giving https://contribute.freecodecamp.org/#/ a read, it details a workflow you can use to avoid these kinds of problems.

**Comment by borntofrappe** [comment]
Thank you @ojeytonwilliams 

I read the guidelines, but it seems I need to give it a second look 👍

**Comment by ojeytonwilliams** [comment]
You're welcome.  

There's definitely a lot to read. If you see anything that's overly verbose or redundant, then streamlining that would help a lot of contributors.

**Pull request merged**